app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/ContributorsFragment.java

```diff
@@ -124,8 +121,36 @@ private void loadContributors(boolean refresh) {
     private static class Adapter
             extends ArrayRecyclerAdapter<ContributorViewModel, BindingHolder<ViewContributorCellBinding>> {
 
-        public Adapter(@NonNull Context context) {
-            super(context);
+        public Adapter(@NonNull Context context, @NonNull ObservableList<ContributorViewModel> list) {
+            super(context, list);
+
+            list.addOnListChangedCallback(new ObservableList.OnListChangedCallback<ObservableList<ContributorViewModel>>() {
+                @Override
+                public void onChanged(ObservableList<ContributorViewModel> contributorViewModels) {
+                    notifyDataSetChanged();
+                }
+
+                @Override
+                public void onItemRangeChanged(ObservableList<ContributorViewModel> contributorViewModels, int i, int i1) {
+                    notifyItemRangeChanged(i, i1);
+                }
+
+                @Override
+                public void onItemRangeInserted(ObservableList<ContributorViewModel> contributorViewModels, int i, int i1) {
+                    notifyItemRangeInserted(i, i1);
+                }
+
+                @Override
+                public void onItemRangeMoved(ObservableList<ContributorViewModel> contributorViewModels, int i, int i1,
+                        int i2) {
+                    notifyDataSetChanged();
```

Comment:
I'm not sure whether notifyItemMoved and onItemRangeMoved are same but it seems to notifyItemMoved works well for our case.